### ISSUE TITLE
Penrun does not propagate SIGINT to child process

### ISSUE THREAD

**Issue by peckto** [Fraunhofer-AISEC/penlog#38]
When terminating penrun with `^C` (`KeyboardInterrupt`), the child process seems to be terminated without propagating the `KeyboardInterrupt` signal.
Wen running a program with `penrun` I would expect the same termination behaviour than running the program right from the shell.

POC
```python
#!/usr/bin/env python3
import time
from penlog import Logger


logger = Logger(component='poc', flush=True)

def test():
    while True:
        logger.log_info('HI')
        time.sleep(1)

try:
    test()
except KeyboardInterrupt:
    logger.log_error('KeyboardInterrupt')
```

Result:
```
$ penrun ./poc.py 
Feb 10 16:25:01.146 {poc     } [message ]: HI
Feb 10 16:25:02.148 {poc     } [message ]: HI
^C$
```

Expected result:
```
$ ./poc.py 
Feb 02 16:27:29.945: HI
Feb 02 16:27:30.946: HI
^CFeb 02 16:27:31.665: KeyboardInterrupt
$
```

**Comment by rumpelsepp** [comment]
The subshell seems to block the signal delivery. I might add a trap in the subshell and forward the signal.

**Comment by rumpelsepp** [comment]
Was a hr bug. Fixed by 63099155aa37df846e3db8ed0a899371b0eb2d42 and 2ccacea6bda41d26ae2e43daa1202ec08b092521
